pkg/testcore/suites/functional-suites.go

```diff
@@ -835,6 +842,12 @@ func (cts *CapacityTrackingSuite) Run(ctx context.Context, storageClass string,
 	if sc.HasError() {
 		return delFunc, sc.GetError()
 	}
+
+	if cts.Image == "" {
+		log.Info("Using default image")
```

Comment:
Can we enhance this log by stating that 'docker.io/centos:latest' is the default image?